extensions/keycloak-authorization/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,14 @@
+
+{
+    "name": "Keycloak Authorization",
+    "labels": [
+        "oauth2",
+        "openid-connect",
+        "keycloak",
+        "authorization-services",
+        "policy-enforcer",
+        "fine-grained-permission",
+        "resource-based-authorization"
+    ],
+    "guide": "https://quarkus.io/guides/keycloak-authorization-guide"
+}
```

Comment:
Please also add it in the legacy json for now: ./devtools/platform-descriptor-legacy/src/main/filtered/extensions.json

Reply:
It is there already.